code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -331,6 +341,13 @@
 			recording_recipe = null
 			. = TRUE
 
+		if("reaction_lookup")
+			//drink dispensers shouldnt be able to look up reagents
```

Comment:
Add this as a variable to `/obj/machinery/chem_dispenser` and override it on drink dispensors. You can then pass this variable inside of ui_static_data, and check it on the TGUI client side by doing {!!can_lookup && <LookupThingyComponent>}

Reply:
Gotcha, came up with b75869b